cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -5894,6 +5894,48 @@
     </description>
   </property>
 
+  <property>
```

Comment:
Can we move these defaults to the extension itself? Generally, extension-specific properties are not put in cdap-default.xml.

Reply:
done.